src/rdkafka.h

```diff
@@ -3854,7 +3900,10 @@ rd_kafka_consumer_group_metadata_new (const char *group_id);
  */
 RD_EXPORT rd_kafka_consumer_group_metadata_t *
 rd_kafka_consumer_group_metadata_new_with_genid (const char *group_id,
-                                                 int32_t generation_id);
+                                                 int32_t generation_id,
```

Comment:
Are any of these fields optional/NULLable? If so, should be reflected in doc string.

Reply:
good idea, `group_instance_id` is nullable.